content/04.results.md

```diff
@@ -48,21 +48,30 @@ The plot on the left (A) is a point range plot of the odds ratio with respect to
+](https://raw.githubusercontent.com/danich1/annorxiver/f5d6c2d04e2fef0d38eaee77ef6c2a0b19ef6358/biorxiv/corpora_comparison/output/figures/preprint_published_comparison.png){#fig:biorxiv_pmc_pre_published_comp}
 
 A preprint's linguistic style can change once a preprint has undergone the revision process prior to being published.
 We quantified this linguistic difference by calculating the odds ratio of tokens appearing in the union in bioRxiv preprints and their published counterparts within PMC.
-Tokens with an odds ratio greater than one are mainly centered on paper/figure references and research specific terms (Figure {@fig: biorxiv_pmc_pre_published_comp}).
-Tokens with an odds ratio of less than one are focused on data availability, and research measurements such as number of cases and controls or significance testing (Figure {@fig: biorxiv_pmc_pre_published_comp}).
+Tokens with an odds ratio greater than one are mainly centered on paper/figure references and research specific terms (Figure {@fig:biorxiv_pmc_pre_published_comp}).
+Tokens with an odds ratio of less than one are focused on data availability, and research measurements such as number of cases and controls or significance testing (Figure {@fig:biorxiv_pmc_pre_published_comp}).
 
-### The bioRxiv Preprint Landscape
-1. Provide the tSNE figure of the bioRxiv 
-2. Discuss the results of the tSNE figure and highlight that there are category clusters within the figure
+### Topic Analysis of bioRxiv's Principal Components
 
-### Topic Analysis of Principal Components
-1. Provide an example of the word cloud for principal components
-2. Show plot of the principal components and the scatterplot
-3. Mention that the word clouds can be found at xyz
+![
+The top two principal components (PCs) consist of the following concepts: molecular vs quantitative biology (PC1) and genomics vs bioinformatics (PC2).
+The left most plot (A) is a word cloud for the second PC.
+Tokens in orange are associated with a PC's positive axis while tokens in blue are associated with the PC's negative axis.
+The size of each token indicates the strength of association.
+The word cloud at the bottom right (C) is for the first PC and follows the same trend as previously described.
+The scatter plot at the top right (B) is a visualization of documents being plotted along the PC's axes.
+Documents were hand-picked based on the concepts captured by each PC.
+](https://raw.githubusercontent.com/greenelab/annorxiver/35d3ea0de3c9c78e3c524736bbaada00928c88fb/biorxiv/pca_association_experiment/output/pca_plots/figures/pca01_v_pca02_figure.png){#fig:pca_word_cloud_plot}
+
+The most predominant source of variation within bioRxiv preprints is the concept of molecular biology vs quantitative biology (Figure {@fig:pca_word_cloud_plot) C).
+This highlights the bisection of biomedical research where majority of results came from molecular biology experiments or quantitative biology experiments.
```

Comment:
Do you want to add panels that show the average (+/- 95% CIs) for eigenvalues for preprints from each self-identified category on bioRxiv with respect to PCs 1 and 2?